internal/zinc-scripted/src/main/contraband/scripted.contra

```diff
@@ -0,0 +1,14 @@
+package sbt.internal.inctest
```

Comment:
I just need some way of describing a build-like structure like https://github.com/sbt/zinc/blob/7185554f6a17839cc2f03d6b853ebca36c99a05b/zinc/src/sbt-test/source-dependencies/binary/build.json